pkg/client/clientset/versioned/clientset.go

```diff
@@ -28,8 +28,6 @@ import (
 type Interface interface {
 	Discovery() discovery.DiscoveryInterface
 	LitmuschaosV1alpha1() litmuschaosv1alpha1.LitmuschaosV1alpha1Interface
-	// Deprecated: please explicitly pick a version if possible.
-	Litmuschaos() litmuschaosv1alpha1.LitmuschaosV1alpha1Interface
```

Comment:
this is done as part of auto-generation?

Reply:
This is deprecated as part of an operator update, Now we have to define explicitly define the version!!
Will switch to  Litmuschaos() from LitmuschaosV1alpha1() soon...